### PR TITLE
Rb tabbed interface

### PR DESCRIPTION
**Pull request by AnnaVi11arrea1**

### CODE REVIEW

**Review by armstrca — CHANGES_REQUESTED** [review]
"Liked photos" needs to match the layout of Users#show

File: app/views/users/show.html.erb
Comment (armstrca) [inline app/views/users/show.html.erb:1]: Line 2 should be:

```
<div class="col-md-6 offset-3">
```

File: app/views/likes/likes.html.erb
Comment (armstrca) [inline app/views/likes/likes.html.erb:1]: Line 2 should be:

```
<div class="col-md-6 offset-3">
```

**Review by armstrca — CHANGES_REQUESTED** [review]

File: app/views/likes/likes.html.erb
Comment (AnnaVi11arrea1) [reply app/views/likes/likes.html.erb:1]: done

File: app/views/users/show.html.erb
Comment (AnnaVi11arrea1) [reply app/views/users/show.html.erb:1]: done!

**Comment by viviancreates** [comment]
Looks good, great job implementing changes requested

File: app/views/users/likes.html.erb
Comment (armstrca) [inline app/views/users/likes.html.erb:1]: This view is kinda broken:

Screenshot 2024-10-28 094723

**Review by armstrca — CHANGES_REQUESTED** [review]

File: app/views/users/likes.html.erb
Comment (AnnaVi11arrea1) [reply app/views/users/likes.html.erb:1]: Changes made to likes view

**Review by armstrca — APPROVED** [review]
d4798a46f283496f96a831800feff102 1

**Pull request merged**